Packs/FeedInfobloxThreatIntelligence/Integrations/FeedInfobloxThreatIntelligence/FeedInfobloxThreatIntelligence.py

```diff
@@ -0,0 +1,946 @@
+        xsoar_type = FeedIndicatorType.Domain
+        value = indicator_data_dict.get("domain") or indicator_data_dict.get("host")
+    elif indicator_type == "IP":
+        xsoar_type = FeedIndicatorType.IP
+        value = indicator_data_dict.get("ip")
+    elif indicator_type == "URL":
+        xsoar_type = FeedIndicatorType.URL
+        value = indicator_data_dict.get("url")
+    elif indicator_type == "EMAIL":
+        xsoar_type = FeedIndicatorType.Email
+        value = indicator_data_dict.get("email")
+    elif indicator_type == "HASH":
+        xsoar_type = FeedIndicatorType.File
+        value = indicator_data_dict.get("hash")
+    else:
+        xsoar_type = FeedIndicatorType.Domain  # Default to domain
+        value = indicator_data_dict.get("host") or indicator_data_dict.get("domain")
+
+    return xsoar_type, value
+
+
+def extract_indicator_fields(
+    indicator_data: dict[str, Any], feed_tags: list[str] = None, tlp_color: str = None
+) -> dict[str, Any]:
+    """Extract fields from Infoblox indicator data based on mapping.
+
+    Args:
+        indicator_data (Dict[str, Any]): Indicator data from Infoblox.
+        feed_tags (List[str], optional): Tags to add to the indicator. Defaults to None.
+        tlp_color (str, optional): TLP color to set. Defaults to None.
+
+    Returns:
+        Dict[str, Any]: Mapped fields for the XSOAR indicator.
+    """
+    fields = {}
+
+    for xsoar_field, mapping in INFOBLOX_FIELD_MAPPING.items():
+        if mapping.get("path"):
+            # Navigate nested fields using dot notation
+            field_value: Any = indicator_data
+            for path_part in mapping["path"].split("."):
+                if field_value and isinstance(field_value, dict) and path_part in field_value:
+                    field_value = field_value.get(path_part)
+                else:
+                    field_value = None
+                    break
+
+            if field_value is not None:
+                # Convert field types
+                if mapping.get("type") == "date" and field_value:
+                    fields[xsoar_field] = field_value
+                elif mapping.get("type") == "int":
+                    try:
+                        fields[xsoar_field] = int(field_value)
+                    except (ValueError, TypeError):
+                        pass
+                elif mapping.get("type") == "bool":
+                    fields[xsoar_field] = field_value.lower() == "true"
+                else:
+                    fields[xsoar_field] = field_value
+        elif mapping.get("default") is not None:
+            fields[xsoar_field] = mapping.get("default")
+
+    # Add feed tags
+    if feed_tags:
+        fields["tags"] = feed_tags
+
+    # Set TLP color if provided
+    if tlp_color:
+        fields["trafficlightprotocol"] = tlp_color
+
+    return fields
+
+
+def calculate_dbot_score(indicator_data: dict[str, Any]) -> int:
```

Comment:
The DBot score calculation and its documentation are inconsistent. A score of 1 is described as 'Unknown' in the docstring but 'Good' or 'Unknown' in the tests. Consider aligning with the standard XSOAR DBot score scale.